src/components/Network.js

```diff
@@ -11,6 +11,11 @@ import ReactLoading from 'react-loading';
 import dateFormatter from '../utils/date';
 import colors from '../index.scss';
 
+const SyncStatus = {
+  'in-sync': 'In sync',
```

Comment:
I think we should use more descriptive texts here. For instance: "Synchronized", "This peer is behind us", and "This peer is ahead of us". But, in the public explorer, maybe we should use only "Synchronized" and "Not synchronized" with a question mark next to the "Not synchronized" with either "behind of us" ou "ahead of us".